src/__tests__/railgun-engine.test.ts

```diff
@@ -200,8 +202,8 @@ describe('railgun-engine', function test() {
             ),
             unshield: {
               tokenData: shield.tokenData,
-              toAddress: '0x1234',
-              value: '0x01',
+              toAddress: ethersAddress,
+              value: transaction.unshieldPreimage.value.toString(),
```

Comment:
`0x1234` and `0x01` are clear placeholders, and since we're in a helper function here, the actual values are variable.